test-cargo-miri/Cargo.toml

```diff
@@ -9,6 +9,7 @@ edition = "2018"
 
 [dependencies]
 byteorder = "1.0"
+rlib-dep.path = "rlib-dep"
```

Comment:
Maybe call this crate `issue_1691` for consistency.
Also please avoid using two different syntaxes for path dependencies here.